### PR TITLE
Fix package native

### PR DESCRIPTION
**Pull request by roironn**
_Upgraded rocks db binding version_

---

_Will hopefully solve build issue

### CODE REVIEW

**Comment by ori-moisis** [comment]
same issue as last time:
```
Error: dlopen(/Users/orimo/code/discover_results/dev2/dist/.externals/rocksdb/build/Release/leveldown.node, 1): no suitable image found.  Did find:
	/Users/orimo/code/discover_results/dev2/dist/.externals/rocksdb/build/Release/leveldown.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x03
	/Users/orimo/code/discover_results/dev2/dist/.externals/rocksdb/build/Release/leveldown.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x03
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -356,7 +356,7 @@ workflows:
       - package_cli:
           requires:
             - build
-            - unit_test
+            # - unit_test
```
Comment (ori-moisis) [inline .circleci/config.yml:359]: undo after you finish debugging?

**Review by ori-moisis — APPROVED** [review]
Might want to consider what happens if more than one instance of salto tries to run concurrently (the first run may delete the second run's binary files?)

**Comment by hadard** [comment]
@roironn anything preventing us from merging this and release the OSS?

**Pull request merged**